docs/atoz.md

```diff
@@ -0,0 +1,487 @@
+### Introduction
+
+This is an A to Z guide that will help you get `wetty` up and running on Debian Stable. It covers the key configuration areas by using copy and paste commands to help you install this application and get it securely up and running. It should also provide enough information to allow you to understand and extend that configuration for your personal requirements.
+
+### Required dependencies
+
+You will need the package `build-essential` to be installed. We need this specifically for `node-gyp` to build packages when using `npm` or `yarn` to install packages.
+
+As the `root` user run these commands:
+
+~~~bash
+apt update
+apt install -y build-essential
+~~~
+
+If you do not have root access and just want to check the dependency is installed you can use this command:
+
+~~~bash
+dpkg -s build-essential | grep Status:
+~~~
+
+If the program is installed you will see this result:
+
+~~~bash
+Status: install ok installed
+~~~
+
+### Create a local user account
+
+For this guide, unless specifically stated, you should not use a `root` account to install and run `wetty`. Please use an existing local account or create one now. 
+
+**Note:** Whichever user runs `wetty` should be the same user you wish to authenticate with via `ssh` to keep this simple.
+
+If you need to create a local user account you can run this command: 
+
+**Important note:** replace `username` with a user name of your choosing and create a password when prompted
+
+~~~bash
+adduser --gecos "" username
+~~~
+
+Switch to your local user now and open an `ssh` session to continue with this guide.
+
+### Install node locally
+
+To install and manage `node` as a local user we are going to use [Node Version Manager](https://github.com/nvm-sh/nvm) as an established solution to installing and managing multiple versions of node without needing `root` access. We are going to install the `lts` or long term support release of `node` to use with this application.
+
+~~~bash
+bash <(curl -s https://raw.githubusercontent.com/nvm-sh/nvm/master/install.sh) && source ~/.profile
```

Comment:
plz point out the danger of piping curl to bash

Reply:
I can change it to a more conventional download then run method or is the warning sufficient? I just took [this](https://github.com/nvm-sh/nvm#install--update-script) method and changed it slightly.

For example, something like this.

'''
wget -qO ~/nvm.sh https://raw.githubusercontent.com/nvm-sh/nvm/master
bash ~/nvm.sh
'''